source/include/ota_mqtt_interface.h

```diff
@@ -97,13 +103,13 @@ typedef OtaErr_t ( * OtaMqttUnsubscribe_t )  ( const char * pTopicFilter,
  *
  * @param[ucQoS]                Quality of Service
  *
- * @return                      OTA_OS_ERR_OK if success , other error code on failure.
+ * @return                      OtaMqttSuccess if success , other error code on failure.
  */
-typedef OtaErr_t ( * OtaMqttPublish_t )( const char * const pacTopic,
-                                         uint16_t usTopicLen,
-                                         const char * pcMsg,
-                                         uint32_t ulMsgSize,
-                                         uint8_t ucQos );
+typedef OtaMqttStatus_t ( * OtaMqttPublish_t )( const char * const pacTopic,
```

Comment:
Should update these parameters according to latest naming conventions

Reply:
let's do that in a separate PR.